src/BackgroundImage.js

```diff
@@ -1,26 +1,29 @@
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
+import { width, responsiveStyle } from 'styled-system'
 
-const image = props => props.image
-  ? { backgroundImage: `url(${props.image})` }
-  : null
+const image = props =>
+  props.image ? { backgroundImage: `url(${props.image})` } : null
 
-const height = props => props.height
-  ? { height: props.height }
-  : null
+const height = props => (props.height ? { height: props.height } : null)
 
 const BackgroundImage = styled.div`
-  ${image}
-  ${height}
-  background-color: ${props => props.theme.colors.gray};
+  ${image} ${height} ${width} background-color: ${props =>
+      props.theme.colors.gray};
   background-position: center;
   background-size: cover;
   background-repeat: no-repeat;
 `
+const numberStringOrArray = PropTypes.oneOfType([
+  PropTypes.number,
+  PropTypes.string,
+  PropTypes.array
+])
 
 BackgroundImage.propTypes = {
   /** background-image url */
-  image: PropTypes.string
+  image: PropTypes.string,
+  width: numberStringOrArray
```

Comment:
You can import the propTypes needed for any of the styled-system functions instead of defining them manually, see the Divider component for an example: https://github.com/pricelinelabs/design-system/blob/master/src/Divider.js#L19-L23

Reply:
done